nltk/tokenize/destructive.py

```diff
@@ -113,6 +114,15 @@ class NLTKWordTokenizer(TokenizerI):
     CONTRACTIONS3 = list(map(re.compile, _contractions.CONTRACTIONS3))
 
     def tokenize(self, text, convert_parentheses=False, return_str=False):
+
+        if return_str is not False:
+            warnings.warn(
+                "Parameter 'return_str' has been deprecated and should no"
+                "longer be used.",
```

Comment:
```suggestion
                "Parameter 'return_str' has been deprecated and should no "
                "longer be used.",
```
This string will be concatenated with no space between `no` and `longer`.